docs/addons/container-insights.md

```diff
@@ -21,3 +28,121 @@ new EksBlueprint(app, 'my-stack-name', addOns, [], {
 });
 ```
 
+Next run `cdk deploy` to update your CDK stack. 
+
+## Prerequisites
+
+Once the Container Insights add-on has been installed to your cluster check to see that the CloudWatch Agent and the FluentD daemons are running. Run `kubectl get all -n amazon-cloudwatch` and you should see the following output.
+```
+NAME                           READY   STATUS    RESTARTS   AGE
+pod/cloudwatch-agent-k8wxl     1/1     Running   0          105s
+pod/fluentd-cloudwatch-78zv4   1/1     Running   0          105s
+
+NAME                                DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/cloudwatch-agent     1         1         1       1            1           <none>          107s
+daemonset.apps/fluentd-cloudwatch   1         1         1       1            1           <none>          106s
+```
+
+To enable or disable control plane logs with the console, please run the following command in your terminal - `aws eks update-cluster-config \
```

Comment:
we should add this by default via the addon. And perhaps a option to disable via addon props.

Reply:
Not needed for this PR though. I have filed a ticket. https://github.com/aws-quickstart/quickstart-ssp-amazon-eks/issues/79

Reply:
+1 sounds good